operators/ior_presets.py

```diff
@@ -0,0 +1,181 @@
+""" BlendLuxCore operator and data classes to support the LuxCoreNodeIORPreset
+    class in module BlendLuxCore/nodes/textures/iorpreset"""
+# <pep8 compliant>
+import copy
+import bpy
+from bpy.props import EnumProperty, StringProperty
+
+# Refractive Index value references:
+#   https://refractiveindex.info/
+#   https://en.wikipedia.org/wiki/List_of_refractive_indices
+#   https://en.wikipedia.org/wiki/Calcite
+
+
+class LuxCoreIORPresetValues():
+    """Class to contain IOR preset items, and return them in
+       a list of tuples suitable for an EnumProperty"""
+
+    # ior_values format: 'ior_values["unique_id"] = ["name", value]'
+    # The ior_values items can be listed here in any order, as they will
+    # be sorted when they are converted to a list as required by EnumProperty.
+    _ior_values = {}
```

Comment:
You could format this dict differently, like so:

```
_ior_values = {
    "acetone": ["Acetone", 1.36],
    "acrylic_glass": ["Acrylic Glass", 1.49],
    ...
}
```
But you don't have to (it's probably annoying, tedious work to reformat now). Just for the future.